python/ray/_private/services.py

```diff
@@ -186,7 +186,8 @@ def get_address_info_from_redis_helper(redis_address,
         client_node_ip_address = client_info["NodeManagerAddress"]
         if (client_node_ip_address == node_ip_address
                 or (client_node_ip_address == "127.0.0.1"
-                    and redis_ip_address == get_node_ip_address())):
+                    and redis_ip_address == get_node_ip_address())
```

Comment:
Come to think of it, do we actually want `get_node_ip_address(redis_address)` here rather than leaving it as the default 8.8.8.8? If the GlobalState will be reporting the IP addresses that it sees from the perspective of `redis_address` (which might not always be the same as the view from 8.8.8.8 Google DNS)?